app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -45,9 +45,12 @@ def textual_bind_addr
   end
 
   def textual_server_state
+    server_state = translated_status(@record.properties['Calculated Server State'] || @record.properties['Server State'])
+    server_state += " (#{translated_status(@record.properties['Suspend State'].downcase)})" if @record.properties['Suspend State'] == 'SUSPENDED'
```

Comment:
@abonas, can you please ensure that a displayed power state in format: "Running (Suspended)" is the accepted form?